benchmarking/process_json_to_csv.jl

```diff
@@ -0,0 +1,171 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Processes a collection of solve results in JSON format into a CSV file.
+#
+# The collection of solve results can correspond to multiple parameter settings.
+# We assume that results for each parameter setting are in a separate directory
+# and are named as *_summary.json, containing serialized SolveLogs.
+# The layout of the results is specified by a JSON file of the following format:
+#
+# {"datasets": [
+#   {"name": "name1", "logs_directory": "path1"},
+#   {"name": "name2", "logs_directory": "path2"}
+#  ] }
+#
+# For example, suppose /tmp/layout.json contains the following content:
+# {"datasets": [
+#   {"name": "pdhg,1e-8", "logs_directory": "/tmp/pdhg_1e-8"},
+#   {"name": "pdhg,1e-4", "logs_directory": "/tmp/pdhg_1e-4"},
+#   {"name": "scs,1e-8", "logs_directory": "/tmp/scs_1e-8"},
+#   {"name": "scs,1e-4", "logs_directory": "/tmp/scs_1e-4"}
+#  ] }
+#
+# Then:
+# $ julia --project=. process_json_to_csv.jl /tmp/layout.json /tmp/dataset.csv
+#
+# will generate a result file in /tmp/dataset.csv containing the summary of each
+# solve in all of four directories, with columns identifying the corresponding
```

Comment:
"with columns" -> "with a column" (unless I misunderstood the setup).

Reply:
Done